MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -2524,15 +2523,13 @@ private void spr()
             //If the supplied string has any control characters for formatting, process them
             var formattedMessage = FormatPrintf(output, 2);
 
-            if (formattedMessage.Length > 0x400)
+            if (formattedMessage.Length > 1024)
```

Comment:
could probably lower this limit, but then again we're not overly concerned with memory usage at this point